src/main/resources/config.yml

```diff
@@ -52,7 +52,9 @@ currency:
     format: "#,##0.00"
 
 # Starting balance
-starting-balance: 0
+# Min value is 0. Set to false to disable creating an account on join.
+# If the Vault integration is enabled and this is set to `null`, an account with 0 balance will be created regardless.
```

Comment:
```suggestion
# If the Vault integration is enabled and this is set to `false`, an account with 0 balance will be created regardless.
```

Reply:
Decided to go without the `` in case people copy that as well. The plugin however will recognise if `"false"` is written as string in quotes and will work correctly